src/services/blocksService.ts

```diff
@@ -19,6 +24,27 @@ export async function getBlock(network: string, filter: BlocksFilter) {
 	return getArchiveBlock(network, filter);
 }
 
+export async function getBlocks(
+	network: string,
+	filter: BlocksFilter|undefined,
+	order: BlocksOrder = "id_DESC",
+	pagination: PaginationOptions,
+) {
+	if (hasSupport(network, "explorer-squid")) {
+		return getExplorerSquidBlocks(network, filter, order, pagination);
+	}
+
+	return {
```

Comment:
Tady bych udelal druhou moznost to tahat z archivu (pokud nema explorer squid), pripadne to tahal jen z achivu. Melo by to fungovat oboji asi stejne. Je to obdobny jako u extrinsicu.